### PR TITLE
Bug 1774864: Enable metrics for collector when LF enabled

### PR DESCRIPTION
**Pull request by jcantrill**
https://bugzilla.redhat.com/show_bug.cgi?id=1774864

### CODE REVIEW

File: pkg/k8shandler/collection.go [hunk 1]
```diff
@@ -149,6 +153,51 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateCollectionPriorityCla
 	return nil
 }
 
+func (clusterRequest *ClusterLoggingRequest) createCollectorMetricsRBAC() error {
+	clusterRole, err := clusterRequest.CreateClusterRole(
+		"log-collector-metrics",
```
Comment (JAORMX) [inline pkg/k8shandler/collection.go]: shouldn't this be created as part of the manifests that deploy the operator?
Giving the operator permission to create clusterroles doesn't seem like a good idea :confused:

Comment (jcantrill) [reply pkg/k8shandler/collection.go]: learned that OLM supports creation of resources from manifest files. modified accordingly

Comment (JAORMX) [reply pkg/k8shandler/collection.go]: Thanks! that's way better!

**Comment by JAORMX** [comment]
uhm.... I wonder if the new cluster role is  being picked up at all by the e2e job.

**Comment by JAORMX** [comment]
@jcantrill you also need to add it to the clusterserviceversion https://github.com/openshift/cluster-logging-operator/blob/master/manifests/4.3/cluster-logging.v4.3.0.clusterserviceversion.yaml

**Comment by JAORMX** [comment]
Here specifically https://github.com/openshift/cluster-logging-operator/blob/master/manifests/4.3/cluster-logging.v4.3.0.clusterserviceversion.yaml#L224

**Comment by jcantrill** [comment]
> Here specifically https://github.com/openshift/cluster-logging-operator/blob/master/manifests/4.3/cluster-logging.v4.3.0.clusterserviceversion.yaml#L224

I was told adding to the CSV will not allow creation off the binding to the SA in a different namespace

**Pull request merged**